hack/bin/deps.bzl

```diff
@@ -236,19 +236,35 @@ def install_kind():
         urls = ["https://github.com/kubernetes-sigs/kind/releases/download/v0.11.1/kind-linux-amd64"],
     )
 
-
-# ytt is a yaml interpolator from the Carvel toolchain https://carvel.dev/.
-def install_ytt():
-    http_file(
-        name = "ytt_darwin",
-        executable = 1,
-        sha256 = "a874395924e670f2c89160efeffc35b94a9bcf4e515e49935cb1ceb22be7f08a",
-        urls = ["https://github.com/vmware-tanzu/carvel-ytt/releases/download/v0.34.0/ytt-darwin-amd64"],
+def install_istioctl():
+    http_archive(
+        name = "istioctl_darwin",
+        sha256 = "3ac12c28f39c49f8e812453cafd5741a39353bb80f92f8b335957d5f9415e04e",
+        urls = ["https://github.com/istio/istio/releases/download/1.10.2/istio-1.10.2-osx.tar.gz"],
+        build_file_content =
+            """
+filegroup(
+    name = "file",
+    srcs = [
+        "istio-1.10.2/bin/istioctl",
+    ],
+    visibility = ["//visibility:public"],
+)
+    """,
     )
 
-    http_file(
```

Comment:
ytt was added as a part of upgrade test PR- [it's needed for the upgrade test](https://github.com/jetstack/cert-manager/blob/master/hack/verify-upgrade.sh#L32). @maelvls I think this PR does not have the latest changes in master, I am confused why parts of the uprade PR are being removed here whilst other parts seem to not be in this PR. Could we rebase this PR off the latest master?

Reply:
Apologies, I might have done a poor job at rebasing; I'll try again.

Reply:
Done! Sorry again, rebasing requires so much brain power 😞